xl2times/transforms.py

```diff
@@ -1791,7 +1793,7 @@ def generate_dummy_processes(
         for col, value in additional_cols.items():
             process_declarations[col] = value
 
-        process_data_specs = process_declarations[["process", "description"]]
+        process_data_specs = process_declarations[["process"]].copy()
```

Comment:
Yep, I think you need to do a copy here since `process_declarations` continues to be used later on and you don't want to add the `ACTCOST` column to it 2 lines later.